tests/functional/codegen/test_interfaces.py

```diff
@@ -171,7 +169,32 @@ def bar() -> uint256:
 def test_extract_file_interface_imports(code, filename, make_input_bundle):
     input_bundle = make_input_bundle({filename: ""})
 
-    assert compile_code(code, input_bundle=input_bundle) is not None
+    assert (
+        compile_code(
+            code, resolved_path=input_bundle.search_paths[0] / filename, input_bundle=input_bundle
+        )
+        is not None
+    )
+
+
+VALID_RELATIVE_IMPORT_CODE = [
+    # import statement, import path without suffix
+    ("from .a import Foo", "a"),
+    ("from ..a import Foo", "a/b"),
+]
+
+
+# TODO CMC 2024-10-13: should probably be in syntax tests
+@pytest.mark.parametrize("code,subdirs", VALID_RELATIVE_IMPORT_CODE)
+def test_extract_file_interface_relative_imports(code, subdirs, make_input_bundle):
+    input_bundle = make_input_bundle({"a/Foo.vyi": ""})
+
+    assert (
+        compile_code(
+            code, resolved_path=(input_bundle.search_paths[0]) / subdirs, input_bundle=input_bundle
```

Comment:
shouldn't the resolved path end with a file? like `code` for the 2nd case should be in `a/b`, right? in which case `..a` resolves to `a/a` and we should get file not found?